src/app/application.cpp

```diff
@@ -405,15 +405,62 @@ void Application::processParams(const QStringList &params)
         return;
     }
 #endif
+    BitTorrent::AddTorrentParams torrentParams;
+    TriStateBool skipTorrentDialog = TriStateBool::Undefined;
 
     foreach (QString param, params) {
         param = param.trimmed();
+
+        // Process strings indicating options specified by the user.
+        
+        if (param.startsWith("@savePath=")) {
+            torrentParams.savePath = param.mid(10);
+            continue;
```

Comment:
It's not only my point of view.
The normal continuation/completion of the loop should be described in the appropriate place (as loop condition). `break` and `continue` instructions are for non-standard/emergency interruption of the loop (or loop iteration). Or, in exceptional cases, to avoid too much code (when other methods do not help or are ineffective).
I don't think that in the case of the use of `else if` this code will become less readable. Besides, it will become three times shorter!